app/controllers/submissions_controller.rb

```diff
@@ -3,6 +3,7 @@ class SubmissionsController < BasePublicController
   include ClaimSubmission
 
   skip_before_action :send_unstarted_claimants_to_the_start, only: [:show]
+  skip_before_action :check_whether_closed_for_submissions, only: [:show]
```

Comment:
This callback is skipped to avoid passing around the `service_access_code` as we clear the session after creating the claim. Additionally I think it makes sense to skip this for the `show` action (but not `create` obv) to handle the, unlikely, scenario where someone submits a claim and we close the journey before they're redirected to the show page, probably best to let them know their claim was submitted.